docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -52,13 +47,13 @@ SOURCE_GALERA_MEMBERS=(
 SOURCE_DB_ROOT_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' MysqlRootPassword:' | awk -F ': ' '{ print $2; }')
 ----
 
-* Prepare MariaDB copy directory and the adoption helper pod
+* Prepare the MariaDB copy directory and the adoption helper pod:
```

Comment:
Is the colon needed here at the end of line 50?

Reply:
Yes, it's introducing some steps.